game.py

```diff
@@ -20,7 +20,24 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    print("* LETS PLAY SNOWMAN ! *")
+    wrong_guesses_list = []
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES:
+        user_input = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+        if user_input in snowman_word:
+            print("You guessed a letter that's in the word!")
+            correct_letter_guess_statuses[user_input] = True
+            print_word_progress_string(snowman_word, correct_letter_guess_statuses)
+        else: 
+            print (f"The letter {user_input} is not in the word")
+            wrong_guesses_list.append(user_input)
+            print_snowman_graphic(len(wrong_guesses_list))
+        
+        if is_word_guessed(snowman_word, correct_letter_guess_statuses) == True:
+            print("Congratulations, you win!")
+            return 
```

Comment:
A `return` statement in Python stops the execution of code. I'm wondering if you can think of another way the logic could be written so that you can end the game without needing to use `return` (or a `break` statement) to break the loop?

This isn't a requirement, but if you'd like more practice with writing conditional logic in Python, I'd suggest thinking through and re-writing the logic so you don't need `return` to stop the game.

If you choose to do so, there's no need for you to submit anything to me for re-review. This would just be for your own practice 😊